apps/expo/src/app/(tabs)/index.tsx

```diff
@@ -1,20 +1,29 @@
 import React from 'react'
-import { Text, View } from 'react-native'
-import { Stack } from 'expo-router'
+import { Text } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
+import { Stack } from 'expo-router'
+
+import { useGlobalContext } from '~/context/global-context'
+
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/c7e2a95d-a2d6-42b0-933c-a22787354379)

lgtm, as we discussied